packages/libs/codemodel/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@autorest/codemodel",
-  "version": "4.14.0",
+  "version": "4.14.1",
```

Comment:
In the past we had discussed keeping codemodel's versioning in line with the versioning of modelerfour so that someone who is using a particular version of modelerfour can use the equivalent major/minor codemodel version (patch isn't important).  Maybe we can bump it to 4.15.1 (in a separate PR?)

Reply:
Hhm, so we can use rush versioning strategy too for those. I think maybe after we move all related packages we can visit what version we want to have